### PR TITLE
set wal_level config not on empty parameters map

### PR DESCRIPTION
**Pull request by FxKu**
fix regression introduced by #2187

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

File: pkg/cluster/sync.go [hunk 1]
```diff
@@ -403,7 +403,10 @@ func (c *Cluster) syncStatefulSet() error {
 		c.logger.Warnf("could not get list of pods to apply PostgreSQL parameters only to be set via Patroni API: %v", err)
 	}
 
-	requiredPgParameters := c.Spec.Parameters
+	requiredPgParameters := make(map[string]string)
+	for k, v := range c.Spec.Parameters {
+		requiredPgParameters[k] = v
+	}
```
Comment (dmvolod) [inline pkg/cluster/sync.go:409]: Probably, `maps.Clone` can be used instead as 
`requiredPgParameters := maps.Clone(c.Spec.Parameters)`

Comment (dmvolod) [reply pkg/cluster/sync.go:409]: Oh, `maps.Copy` should be used in this case only

**Comment by idanovinda** [comment]
👍

**Pull request merged**